src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -391,8 +391,17 @@ def safe_to_delete(self, svc_ids):
 
     @RESTController.Resource('GET')
     def devices(self, svc_id):
+        devices: list = CephService.send_command(
```

Comment:
Having a look at it, it can return both a `str` or [anything returned by `json.loads`](https://github.com/python/typing/issues/182):
https://github.com/ceph/ceph/blob/14189b78d81fdfa5c81b6674b8e5ee963f92c0e6/src/pybind/mgr/dashboard/services/ceph_service.py#L194-L230